pipenv/vendor/tomlkit/items.py

```diff
@@ -6,7 +6,10 @@
 from datetime import date
 from datetime import datetime
 from datetime import time
-from enum import Enum
+try:
+    from enum import Enum
+except ImportError:
+    from pipenv.vendor.backports.enum import Enum
```

Comment:
This uses try-catch, but there’s a `lru_cache` block that uses if-else. I would recommend unifying the style (probably to try-catch; it seems to be recommended).

Reply:
`lru_cache` is got from the original code, I will change to `if` to adapt the style.

Reply:
in _most_ cases, like the `Enum` case, it doesn't matter at all